### DISCUSSION TITLE
專為超新手精心設計的自學課程

### DISCUSSION THREAD

**Discussion by Doreen331** [1101-GS4719-C/Python-Community-GS4719-C#960] [General]
昨天在臉書社團看到這個作者推廣他自編的教材，沒想到今天在Dcard又再看到一次，點進去後發現作者真的超用心，而且內容真的淺顯易懂，十分適合超級新手。
對我個人而言最特別的是RWD的部分，之前寫網頁的時候以為RWD只是去bootstrap上抓連結複製貼上而已，但這個網站裡有更詳細的介紹，讓我對響應式有更多了解，真的高手在人間，希望作者可以持續更新！

https://codelove.tw/courses

**Comment by swerfg03joe** [comment]
本身認為寫HTML這類的標籤語言，都是有固定的套路，另外製作網站也需要有一定程度的美感，否則空有功能的網頁也無法吸引人流

**Comment by BlahFish1006** [comment]
哇！謝謝分享~ 能感覺到作者真的十分用心，甚至還有可以提問討論的交流區，對問題一堆的新手實在非常之友善。

**Comment by Tuoresa** [comment]
哇阿阿阿這也太用心！
之前有寫網頁的經驗
但那時候基礎知識極少，過程有點像是找到什麼功能就複製它的程式碼
一直很想更加完善那個的網頁，讓它貼近自己的想像
非常感謝分享！希望作者能一直更新下去哈哈

**Comment by dabbie109303501** [comment]
感謝分享 這個對想寫網頁的人幫助很大 裡面提供的網站也很好用
看了版主提到的RWD 查了資料後 很想學習這個技術
因為目前有在寫APP 而各個廠牌的手機螢幕畫面的大小都不一樣
導致原本的版面在一些手機上會變得很奇怪很醜
希望我有能力好好運用到APP上QQ
 
這裡有RWD的詳細介紹https://www.ibest.tw/page01.php
